src/sentry/mail/actions.py

```diff
@@ -38,3 +38,24 @@ def after(self, event, state):
 
     def get_form_instance(self):
         return self.form_cls(self.project, self.data)
+
+
+class NotifyActiveReleaseEmailAction(NotifyEmailAction):
```

Comment:
Introduce a new Action type. In reality, this won't be exposed in the UI, cannot be defined in an issue alert rule, and will only be used in this scenario where we deliberately instantiate it as part of rule-processing one new issues.

Reply:
Can you add a comment here describing what it's used for?